Source/WebKit/UIProcess/SpeechRecognitionPermissionManager.cpp

```diff
@@ -189,7 +189,7 @@ void SpeechRecognitionPermissionManager::continueProcessingRequest()
 void SpeechRecognitionPermissionManager::completeCurrentRequest(std::optional<WebCore::SpeechRecognitionError>&& error)
 {
     ASSERT(!m_requests.isEmpty());
-    auto [currentRequest, frameInfo] = m_requests.takeFirst();
+    Ref currentRequest = m_requests.takeFirst().first;
```

Comment:
Does this need a `WTFMove` or will it move without us doing anything explicit?

Reply:
That's a good point, it probably doesn't move without an explicit WTFMove() here.